zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -67,15 +67,20 @@ <h3 class="interpreter-title">{{setting.name}}
     </div>
     <br />
     <div class="row interpreter">
-      <div class="col-md-12">
-        <b>Properties</b>
+      <div ng-show="getInterpreterPropertiesCount(setting.id) === 0 || valueform.$hidden" class="col-md-12 empty-properties-message">
```

Comment:
``getInterpreterPropertiesCount(setting.id) === 0``
can be replaced by ``_.isEmpty(setting.properties)`` if you add ``$scope._ = _;`` in the controller.